WPThemeReview/ruleset.xml

```diff
@@ -16,6 +16,11 @@
 	<!-- Alternative PHP open tags not allowed. -->
 	<rule ref="Generic.PHP.DisallowAlternativePHPTags"/>
 
+	<!-- Restrict PHPCS' internal error message for short tag use/no code to PHP files only. -->
+	<rule ref="Internal.NoCodeFound">
+		<include-pattern>*\.(php|inc)$</include-pattern>
```

Comment:
I'm no so sure this the right solution as this is not a sniff ti include/exclude files for, but an PHPCS native error code.

Reply:
Just thinking - it may not even be something to solve here as PHPCS natively doesn't throw this for CSS/JS files. I think this may have something to do with the Tokenizer short-circuit for minified files in the plugin.